circle.yml

```diff
@@ -1,6 +1,6 @@
 machine:
   xcode:
-    version: 8.2
+    version: 8.3
```

Comment:
No idea if we should specify 8.3.2 because of this:
https://discuss.circleci.com/t/we-are-deprecating-the-xcode-8-3-build-image/12115